tap_snapchat_ads/streams.py

```diff
@@ -17,758 +17,1419 @@
+    paging = False
+    parent = 'organization'
+    params = {}
+
+# Reference: https://developers.snapchat.com/api/docs/?python#get-all-billing-centers
+class BillingCenters(SnapchatAds):
+    tap_stream_id = 'billing_centers'
+    parent_stream = 'organizations'
+    key_properties = ['id']
+    replication_method = 'INCREMENTAL'
+    replication_keys = ['updated_at']
+    path = 'organizations/{parent_id}/billingcenters'
+    data_key_array = 'billingcenters'
+    data_key_record = 'billingcenter'
+    paging = True
+    parent = 'organization'
+    params = {}
+    
+# Reference: https://developers.snapchat.com/api/docs/?python#get-all-members-of-an-organization
+class Members(SnapchatAds):
+    tap_stream_id = 'members'
+    parent_stream = 'organizations'
+    key_properties = ['id']
+    replication_method = 'INCREMENTAL'
+    replication_keys = ['updated_at']
+    path = 'organizations/{parent_id}/members'
+    data_key_array = 'members'
+    data_key_record = 'member'
+    paging = False
+    parent = 'organization'
+    params = {}
+                
+# Reference: https://developers.snapchat.com/api/docs/?python#get-all-roles-in-organization
+class Roles(SnapchatAds):
+    tap_stream_id = 'roles'
+    parent_stream = 'organizations'
+    key_properties = ['id']
+    replication_method = 'INCREMENTAL'
+    replication_keys = ['updated_at']
+    path = 'organizations/{parent_id}/roles'
+    data_key_array = 'roles'
+    data_key_record = 'role'
+    paging = True
+    parent = 'organization'
+    params = {}
+    
+# Reference: https://developers.snapchat.com/api/docs/#get-all-ad-accounts
+class AdAccounts(SnapchatAds):
+    tap_stream_id = 'ad_accounts'
+    parent_stream = 'organizations'
+    key_properties = ['id']
+    replication_method = 'INCREMENTAL'
+    replication_keys = ['updated_at']
+    path = 'organizations/{parent_id}/adaccounts'
+    data_key_array = 'adaccounts'
+    data_key_record = 'adaccount'
+    paging = False
+    parent = 'organization'
+    params = {}
+    children = ['ad_account_stats_daily', 'ad_account_stats_hourly', 'audience_segments', 'pixels', 'media', 'creatives', 'phone_numbers', 'campaigns', 'ad_squads', 'ads']
+    
+# Reference: https://developers.snapchat.com/api/docs/#get-ad-account-stats
+class AdAccountStatsDaily(SnapchatAds):
+    tap_stream_id = 'ad_account_stats_daily'
+    grandparent_stream = 'organizations'
+    parent_stream = 'ad_accounts'
+    json_schema = 'schemas/shared/ad_account_stats.json'
+    key_properties = ['id', 'start_time']
+    replication_method = 'INCREMENTAL' # query filtered
+    replication_keys = ['end_time']
+    bookmark_query_field_from = 'start_time'
+    bookmark_query_field_to = 'end_time'
+    path= 'adaccounts/{parent_id}/stats'
+    data_key_array = 'timeseries_stats'
+    data_key_record = 'timeseries_stat'
+    date_window_size = 30
+    paging = False
+    parent = 'ad_account'
+    params = {
+        # 'test': 'true', # COMMENT OUT
```

Comment:
These kinds of comments are present already but let's remove them as part of refactoring as it looks like debugging comment.

Reply:
Removed unnecessary comments.